tfhe/src/shortint/ciphertext/standard.rs

```diff
@@ -241,11 +249,13 @@ pub(crate) fn unchecked_create_trivial_with_lwe_size(
     pbs_order: PBSOrder,
     ciphertext_modulus: CiphertextModulus,
 ) -> Ciphertext {
```

Comment:
This function could take an `ShortintEncoding` as input
I don't know this is a better though

Reply:
I don't think it's better as to create the ciphertext, the metadata